### ISSUE TITLE
[Bug] Some keyboard shortcuts missing 

### ISSUE THREAD

**Issue by sigaloid** [zadam/trilium#1851]
0.47.0 beta  (desktop install)
![image](https://user-images.githubusercontent.com/69441971/114881128-3e6e6200-9df2-11eb-9d1b-ca4df25dc140.png) (this is the only missing one in the beta)

0.46.7  (on webserver in browser) 
![image](https://user-images.githubusercontent.com/69441971/114881530-a2912600-9df2-11eb-8911-6503a6ea9daf.png)
![image](https://user-images.githubusercontent.com/69441971/114881579-ab81f780-9df2-11eb-955b-1dc57cc473b7.png)

Reproducible, I don't see any errors in the console

**Comment by zadam** [comment]
Hi, thanks for reporting this issue.

I fixed the missing search shortcut. The other ones are not set because they mostly conflict with existing keyboard shortcuts - e.g. `CTRL-T` will open a new browser tab and not a trilium tab. So for web/server there are no default shortcuts, user can of course still set some. I changed the help page to display "not set" when no shortcut is assigned, hopefully it will be a bit better this way.
